plugin/evm/import_tx_test.go

```diff
@@ -178,7 +178,7 @@ func TestImportTxSemanticVerify(t *testing.T) {
 		Outs: []EVMOutput{evmOutput},
 	}
 
-	state, err := vm.chain.BlockState(vm.lastAccepted.ethBlock)
+	state, err := vm.chain.CurrentState()
```

Comment:
Are we sure this is the desired behavior? Unlike geth, we don't have time-consuming PoW, which means the "current" block may not be the block interested (accepted). Need to double check what CurrentState returns.